avae/napari.py

```diff
@@ -46,7 +46,7 @@ def process(
     )
 
     with torch.inference_mode():
-        x = model.decoder(z, pose)
+        x = model.module.decoder(z, pose)
```

Comment:
is this a leftover of the dataparallel implementation?

Reply:
yes , sorry I have missed this one, resolved now